src/client/containers/Channel/Channel.js

```diff
@@ -53,13 +54,29 @@ export default function Channel() {
   if (messages.length === 0) {
     return (
       <>
+        <ChannelHeadNav
+          imgUrl={
+            userFromDatabase.profileImage || 'https://loremflickr.com/320/240'
+          }
+          urlBack="/overview"
+          channelName={userFromDatabase.user_name}
+          channelId={channelId}
```

Comment:
Hi Wael. Please take a look into my PR https://github.com/HackYourFuture-CPH/chattie/pull/239 it contains ChannelHeadNav in channel.js. And userFromDatabase this is the current user, we need information about another user with whom the current user is chatting.